settings/installer/ospd/firewall/rules/default.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I like this being in a separated place, but it will be for packstack as well, not just osp-d, so I think it's not right to put it under ospd